src/scribe-analytics.js

```diff
@@ -1067,23 +1094,23 @@ if (typeof Scribe === 'undefined') {
       this.context.fingerprint = Env.getFingerprint();
 
       this.context.sessionId = (function() {
-        var sessionId = sessionStorage.getItem('scribe_sid') || Util.genGuid();
+        var sessionId = Util.store.session.getItem('scribe_sid') || Util.genGuid();
 
-        sessionStorage.setItem('scribe_sid', sessionId);
+        Util.store.session.setItem('scribe_sid', sessionId);
```

Comment:
See, another option may just be try / catching this. It will be less code (= smaller distribution size).